app/src/main/java/co/jp/duonghb/MainActivity.kt

```diff
@@ -32,14 +32,13 @@ class MainActivity : ComponentActivity() {
 
 @Composable
 fun Greeting(name: String) {
-    Text(text = "Hello $name!")
+     Text(text = "Hello $name!")
 }
 
 @Preview(showBackground = true)
 @Composable
 fun DefaultPreview() {
-
     BitriseCIDemoTheme {
-        Greeting("Android")
+         Greeting("Android")
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nokaduonghb/BitriseCIDemo/blob/ea47182dd9b23a46040da1a46e2c6975ddd11f85/app/src/main/java/co/jp/duonghb/MainActivity.kt#L42"></span>Unexpected indentation (9) (should be 8)</td>
    </tr>
  </tbody>
</table>